internal/provider/provider.go

```diff
@@ -87,7 +87,8 @@ func (p *BloxOneProvider) Resources(ctx context.Context) []func() resource.Resou
 		ipam.NewAddressResource,
 		ipam.NewRangeResource,
 		ipam.NewFixedAddressResource,
-		ipam.NewServerResource,
+    ipam.NewServerResource,
+		ipam.NewHaGroupResource,
```

Comment:
Looks like a formatting issue. Have you run gofmt?

Reply:
Ahh, no actually. I fixed the conflicts using the Github UI.